### PR TITLE
Update Sample Installation Events

### PR DESCRIPTION
**Pull request by TzviLipschitz**
release/2024
PR (merge when 2024 edge on k8s is about to be released)

Develop
PR (merge when 2024 edge on k8s is about to be released)

### CODE REVIEW

**Review by bbyreddy — COMMENTED** [review]
@TzviLipschitz , just for your information, this change is for 2024 release.

File: content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md [hunk 1]
```diff
@@ -87,6 +87,6 @@ Events:
 …………
   Normal   Installing        2m     cumulocityiotedge  finished installing thin-edge
 …………
-  Normal   Ready             1m     cumulocityiotedge  installed successfully
+  Normal   Ready             1m     cumulocityiotedge  Cumulocity IoT Edge installation is complete, and it's now running version 1017.0.0-4638
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md:90]:
```suggestion
  Normal   Ready             1m     cumulocityiotedge  {{< product-c8y-iot >}} Edge installation is complete, and it's now running version 1017.0.0-4638
```

Comment (TzviLipschitz) [reply content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md:90]: @MWindel As this is a code snippet, we do not need the {{< product-c8y-iot >}} shortcode here. This line will always show as "Cumulocity IoT Edge installation is complete, and it's now running version X"

Comment (MWindel) [reply content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md:90]: @TzviLipschitz Thank you for clarifying.

**Review by MWindel — APPROVED** [review]

**Pull request closed without merging**